Adding gender

eleensmathew opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

##  Added Gender drop down option in register form

<!--- Describe your changes in detail -->

Added gender option as dragdown in front end, and saved it to backend.
Unfortunately trying to use {% load index %} in profile.html was not getting reflected, so i copy pasted relevant css from index.html and changed the background image.
 
Profile looks like this right now
![Screenshot 2023-10-12 002226](https://github.com/ContriHUB/Sheher/assets/92302620/11e72af3-6310-48a0-853f-3d80b601c815)

## Related Issue

<!--- This project only accepts pull requests related to open issues -->
<!--- If suggesting a new feature or change, please discuss it in an issue first -->
<!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
<!--- Please link to the issue here: -->

Fixes # (https://github.com/ContriHUB/Sheher/issues/61)

## Type of change

<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

eleensmathew commented:
Also the commit time is shown wrong? Any reasons for this?

kunal2812 commented:
Not sure. Probably depends upon your system clock

kunal2812 commented on templates/home/profile.html:
```diff
@@ -18,7 +19,7 @@
       }
       body:before {
```
Fix background css

eleensmathew replied:
should i put a new picture then or should i revert changes?

kunal2812 replied:
Picture is okay but it should not repeat

This is the problem
![image](https://github.com/ContriHUB/Sheher/assets/59749539/8fbb2572-f7cf-4c6f-8f03-70f1c3232546)

kunal2812 replied:
Try this

```
element-selector {
    background-image: url('your-image-url.jpg');
    background-repeat: no-repeat;
}
```

eleensmathew replied:
![Screenshot 2023-10-12 010102](https://github.com/ContriHUB/Sheher/assets/92302620/b776ed2b-b202-4fe0-bf23-81aa2c3c0a14)
Fixed the background but not sure why navbar css is not loading.

kunal2812 commented:
Also add navbar to profile page

eleensmathew commented:
Any suggestions for fixing css of nav bar? Maybe it has something to do with location of scripts file?

kunal2812 commented:
Yes, it might be due to css and script files not properly linked for that page

eleensmathew commented:
I tried a few ways to fix this but no changes reflected on the page. Do you have any suggestions?

kunal2812 commented:
Will check and let you know

kunal2812 commented:
@eleensmathew You can remove the navbar and add Home, Report, Log Out Buttons just like on the Report page

eleensmathew commented:
I added some extra functionality in navbar_compliants.html. The css was still not getting loaded so I basically tried to hard code the buttons and now it looks like this. Also added an extra status number =3 so the navbar won't get shown twice
![Screenshot 2023-10-12 233808](https://github.com/ContriHUB/Sheher/assets/92302620/5aed2e4d-64ed-41b4-b096-6dc741b78874)

eleensmathew commented:
Let me know if I should I remove this and do something else.

kunal2812 commented:
![image](https://github.com/ContriHUB/Sheher/assets/59749539/4bc1989f-dbd4-4733-ba5c-b64044046c0c)

Getting this error when I click on Profile

kunal2812 commented:
![image](https://github.com/ContriHUB/Sheher/assets/59749539/4ea2d671-3132-4b30-8748-a8d2e2f67670)

Also mail is not getting displayed properly

kunal2812 commented:
> @eleensmathew You can remove the navbar and add Home, Report, Log Out Buttons just like on the Report page

Buttons for Home, Report are still not present in the navbar

eleensmathew commented:
Okay I'll add it now

eleensmathew commented:
![Screenshot 2023-10-13 001636](https://github.com/ContriHUB/Sheher/assets/92302620/eb6905b5-d1c8-4c95-b252-6273037d52ec)
It looks like this now but I'm not sure fow to fix the text problem. I have fixed the profile link and added home and complaint link.

kunal2812 commented:
When you click on those buttons it should not open a new tab. Expected behaviour is to refresh the same page. 
Also try to fix the text issue. Will add bonus.

eleensmathew commented:
Sure will try to fix it.

On Fri, 13 Oct 2023 at 01:54 Kunal Katiyar ***@***.***> wrote:

> When you click on those buttons it should not open a new tab. Expected
> behaviour is to refresh the same page.
> Also try to fix the text issue. Will add bonus.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ContriHUB/Sheher/pull/72#issuecomment-1760317329>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWAG2HAOEBUI3GD64OKYAWDX7BGWXANCNFSM6AAAAAA54OVIYY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
-- 
Eleena

eleensmathew commented:
HI @kunal2812 I fixed the links to open on the same page and got the long email to be broken into the next line. As a bonus i also changed the background color of edit_profile.html because nav bar text and backgroung colour was same and also centered the hedaing. It currently looks like this 
![Screenshot 2023-10-13 150145](https://github.com/ContriHUB/Sheher/assets/92302620/dd5aa7cd-7eec-4d18-ab27-c9e2f0fb4259)
![Screenshot 2023-10-13 150212](https://github.com/ContriHUB/Sheher/assets/92302620/659d18c6-cde5-4035-83c8-b65cd455ddad)

kunal2812 reviewed (approved):
LGTM
